contracts/interfaces/IERC1404.sol

```diff
@@ -0,0 +1,26 @@
+pragma solidity ^0.5.9;
```

Comment:
@afmfe-iul This is the official ERC1404 interface as described in the EIP that can be found [here](https://github.com/ethereum/EIPs/issues/1404). It is not closed but I can't forsee major changes to it so it's safe to assume that it will remain identical for the main things.